index.js

```diff
@@ -67,7 +105,34 @@ function getAverageIMDBRating() {}
  *  countByRating(movies);
  *  //> { G: 3, PG: 7 }
  */
-function countByRating() {}
+
+// the returned object should have the following format: somenewobj[movie.rated] = the number of those ratings. 
+// may have to do multiple if statments to test out each rating. --> there are 3 ratings: 'G', 'PG', "PG-13" by simply consoling logging movie.rated and pushing into an empty array.
+// the default will return an empty obj,
+
```

Comment:
It looks like we have some issues with how approach this.  Currently, we're iterating through the array and then increasing `numOfTimes`  every time we have _any_ match.  Then we set that `numOfTimes` to whatever key was last seen.

Try this - loop through the array and then check to see if _objOfRatings_ has a key of whatever our movie is rated ( there are only to possible outcomes, the object does or it does not).  If the object _does not_ have a key with the value of the movie rating - create it
Otherwise iterate its value by one.